### PR TITLE
SCM-919 jgit codecheck cleanup

### PR DESCRIPTION
**Pull request by cquoss**
Codecheck cleanup for jgit classes, see issue SCM-919

### CODE REVIEW

File: maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/command/JGitTransportConfigCallback.java [hunk 1]
```diff
@@ -25,81 +25,106 @@
...
-                logger.debug("using private key: " + repo.getPrivateKey());
-                sshSessionFactory = new ProtectedPrivateKeyFileSessionFactory(repo);
-            } else {
+    public JGitTransportConfigCallback( GitScmProviderRepository repo, ScmLogger logger )
+    {
+        if ( repo.getFetchInfo().getProtocol().equals( "ssh" ) )
+        {
+            if ( !StringUtils.isEmptyOrNull( repo.getPrivateKey() ) && repo.getPassphrase() == null )
+            {
+                logger.debug( "using private key with passphrase: " + repo.getPrivateKey() );
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/command/JGitTransportConfigCallback.java]: Did you notice that the message is wrong? It says that the key has a passphrase, but the test says `repo.getPassphrase() == null`.

File: maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/command/JGitTransportConfigCallback.java [hunk 2]
```diff
@@ -25,81 +25,106 @@
...
     @Override
-    public void configure(Transport transport) {
-        if (transport instanceof SshTransport) {
-            SshTransport sshTransport = (SshTransport) transport;
-            sshTransport.setSshSessionFactory(sshSessionFactory);
+    public void configure( Transport transport )
+    {
+        if ( transport instanceof SshTransport )
+        {
+            SshTransport sshTransport = ( SshTransport ) transport;
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/command/JGitTransportConfigCallback.java]: Spaces aren't required around casts.

File: maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/command/JGitTransportConfigCallback.java (hunk 1 above)
Comment (cquoss) [reply maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/command/JGitTransportConfigCallback.java]: You're right.  Will switch the messages.

File: maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/command/JGitTransportConfigCallback.java (hunk 2 above)
Comment (cquoss) [reply maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/command/JGitTransportConfigCallback.java]: Will revert that change.

**Comment by jira-importer** [comment]
Resolve #1143

**Pull request closed without merging**